README.md

```diff
@@ -1,3 +1,23 @@
+# Stored Procedure Unit Tester
+
+An appplication to unit test stores procedures among databases
+
+Licenses under [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
+
+# Release Notes
+
+## Version 0.1.0
+
+### New Features
+ * Added support for connecting to a local Postres database
+ * Added a user interface for connecting to a local database
+ * Added a button to display all stored procedures from a local database
+ 
+### Bug Fixes
+ * None
+### Known Issues
+ * Pressing the connect button infinitley establishes a connection with the Postgres database
```

Comment:
Currently, the connection configuration is a hardcoded set of values which is a known issue.